setup.cfg

```
@@ -1,5 +1,62 @@
 [metadata]
-description-file = README.md
+url = https://github.com/nipype/pydra
+author = nipype developers 
+author_email = neuroimaging@python.org
+maintainer = nipype developers
+maintainer_email = neuroimaging@python.org
+description = Pydra dataflow engine
+long_description = file:long_description.rst
+long_description_content_type = text/x-rst; charset=UTF-8
+license = Apache License, 2.0
+provides =
+    pydra
+classifiers =
+    Development Status :: 2 - Pre-Alpha
```

Is the next release going to be alpha? It looks like we're getting there...

```suggestion
    Development Status :: 3 - Alpha
```